test/files/neg/macro-deprecate-idents.check

```diff
@@ -55,7 +55,7 @@ macro-deprecate-idents.scala:3: error: '=' expected but '}' found.
 macro-deprecate-idents.scala:7: error: '=' expected but '}' found.
 }
 ^
-macro-deprecate-idents.scala:42: error: '{' expected but ';' found.
+macro-deprecate-idents.scala:42: error: '{' expected but (inferred) ';' found.
```

Comment:
I'm surprised that so few test cases needed updating. I suppose that in most cases semi inference is disabled when waiting for a matching brace/paren.

WDYT about removing the " but (inferred) ';' found." from the error message altogether? The position of the error communicates the problem clearly.

Reply:
I was also surprised. Yes, `(inferred) ';'` was not felicitous. I like both your suggestions to let the caret speak for itself. More caret than shtick.